driver-core/src/test/java/com/datastax/driver/core/policies/AbstractRetryPolicyIntegrationTest.java

```diff
@@ -113,18 +114,18 @@ protected void simulateError(int hostNumber, Result result) {
     protected void simulateError(int hostNumber, Result result, Config config) {
         PrimingRequestBuilder builder = PrimingRequest.queryBuilder()
                 .withQuery("mock query")
-                .withResult(result);
+                .withThen(then().withResult(result));
 
         if (config != null)
-            builder = builder.withConfig(config);
+            builder = builder.withThen(then().withConfig(config));
```

Comment:
I think changing simulateError to this would fix it:

```java
    protected void simulateError(int hostNumber, Result result, Config config) {
        PrimingRequest.Then.ThenBuilder then = then().withResult(result);
        PrimingRequestBuilder builder = PrimingRequest.queryBuilder().withQuery("mock query");

        if (config != null)
            then = then.withConfig(config);

        builder = builder.withThen(then);

        scassandras.node(hostNumber).primingClient().prime(builder.build());
    }
```